hazelcast/src/test/java/com/hazelcast/internal/util/phonehome/PhoneHomeIntegrationTest.java

```diff
@@ -163,5 +169,81 @@ public void testCacheMetrics() {
                 .withQueryParam("cawact", equalTo("1")));
     }
 
+    @Test
+    public void testMapLatencies() {
+        MapStoreConfig mapStoreConfig = new MapStoreConfig();
+        mapStoreConfig.setEnabled(true);
+        mapStoreConfig.setImplementation(new DelayMapStore());
+        IMap<String, String> iMap = node.hazelcastInstance.getMap("hazelcast");
+        node.getConfig().getMapConfig("hazelcast").setMapStoreConfig(mapStoreConfig);
+        iMap.put("key1", "hazelcast");
+        iMap.put("key2", "phonehome");
+        iMap.get("key3");
+        LocalMapStatsImpl mapStats = (LocalMapStatsImpl) iMap.getLocalMapStats();
+
+
+        stubFor(get(urlPathEqualTo("/ping"))
+                .willReturn(aResponse()
+                        .withStatus(200)));
+
+        phoneHome.phoneHome(false);
+        verify(1, getRequestedFor(urlPathEqualTo("/ping"))
+                .withQueryParam("mpptla",
+                        equalTo(String.valueOf(mapStats.getTotalPutLatency() / mapStats.getPutOperationCount())))
```

Comment:
Please extract `mapStats.getTotalPutLatency() / mapStats.getPutOperationCount()` into a local variable, it is calculated twice (same for the `mapStats.getTotalGetLatency() / mapStats.getGetOperationCount()` expression)